### PR TITLE
use distroless for base image

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>

### Description:

- use distroless for base image
- use trivy for image scan
- "version" property for PR builds are now "PR-XXX" format
- build binaries and publish them when released (only for linux)
- fix info.BuildTime default value

### Related Issue:

nothing.

### How Has This Been Tested?:

nothing.

### Environment:

- Go Version: 1.14.4
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.0

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [X] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,7 @@ go 1.14
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
+	github.com/aws/aws-sdk-go => github.com/aws/aws-sdk-go master
```
Comment (kpango) [inline go.mod:7]: Do we need to use master branch ?
if it's necessary please define this replace in hack/go.mod.default

Comment (rinx) [reply go.mod:7]: The required version to build it correctly is later than v1.30.x. I'll add v1.33.21 to hack/go.mod.default

Comment (rinx) [reply go.mod]: updated! thanks @kpango

**Comment by kpango** [comment]
@rinx Is there any problem if I merge this PR?

**Comment by rinx** [comment]
@kpango Thanks for reviewing. There's no problem.

**Pull request merged**